lib/Database.py

```
@@ -10,7 +10,7 @@ def connect_to_db(fn):
             Database.db = sqlite3.connect(fn, detect_types=sqlite3.PARSE_DECLTYPES|sqlite3.PARSE_COLNAMES)
             Database.cursor = Database.db.cursor()
         except:
-            print("Can not connect to database")
+            pass

```

better to raise an exception on this